### DISCUSSION TITLE
Welcome to Manager Discussions!

### DISCUSSION THREAD

**Discussion by antleblanc** [ovh/manager#4290] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wilfriedko** [comment]
Hello, I cannot delete the wordpress module in my ovh hosting.
I am experiencing this error: An error occurred while requesting to delete the wordpress module. (User not granted for this request).

However, I have the administrator right.
I do not understand.

**Comment by CRduweb** [comment]
Bonjour, je suis hébergé chez OVH et je rencontre un grand problème que j'aimerais résoudre… 

Merci pour votre aide...
